resources/views/question/question_list.blade.php

```diff
@@ -0,0 +1,50 @@
+@extends('layouts.master')
+
+@section('content')
+
+    <!-- Page Content -->
+    <div id="page-wrapper">
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col-lg-12">
+                    <h1 class="page-header">Question
+                        <small>Question List</small>
+                    </h1>
+                </div>
+                <!-- /.col-lg-12 -->
+                <div class="col-lg-7" style="padding-bottom:120px">
+                    <form action="admin/question/add" method="POST">
```

Comment:
sử dụng laravel collective

Reply:
```suggestion
                    <form action="admin/question/add" method="POST">
```                    {!! Form::open(['method' => 'POST', 'url' => 'admin/question/add']) !!}
                           //
                    {!! Form::close() !!}
Em fix tn đúng chưa hả chị, chị xem hộ em với !!!